### ISSUE TITLE
Add Custom InventoriesGUI?

### ISSUE THREAD

**Issue by Goodstuff4UonYT** [yescallop/Nukkit#50]
It would be nice, if you can do this also in Nukkit.
![dgpfktvu0aanusq](https://user-images.githubusercontent.com/19636759/29047893-4d049612-7bce-11e7-80e6-c4eb8f62f728.jpg)

**Comment by ghost** [comment]
@kvetinac97

**Comment by larryTheCoder** [comment]
Looks like steadfast had these things implemented

**Comment by kvetinac97** [comment]
I am already working on ModalFormsAPI, work in progress !!!

**Comment by yescallop** [comment]
i suggest the ModalForm system should have a complete callback system, like player.showModalForm(form, resp -> logger.info(resp.getString("username")));

**Comment by kvetinac97** [comment]
Yes, there will be a PlayerModalFormRespondedEvent with properties
getResponse (ModalFormResponse class, from which you will be able to pick data)
getForm(The instance of original Form responded)
closed (If player closed the form with Cross button)

**Comment by kvetinac97** [comment]
Now you are able to create it with my PR #54

**Comment by Goodstuff4UonYT** [comment]
@kvetinac97 How does setIcon works?

**Comment by Goodstuff4UonYT** [comment]
@kvetinac97 And what did I wrong? :'D `addElement(new ElementButton("test")); `is wrong :c
I used FormWindowCustom.

**Comment by kvetinac97** [comment]
Custom forms do not support buttons - submit button will appear automatically (MC adds it). Simple forms do.

**Comment by Goodstuff4UonYT** [comment]
@kvetinac97 Okey thanks!

**Comment by ghost** [comment]
@yescallop I can help you rewrite 1.2 inventory, watch there: https://gist.github.com/dktapps/dcd68142f18da90cf22a02e99de26576

**Comment by Goodstuff4UonYT** [comment]
@kvetinac97 I added buttons in Simple forms, but the images are not shown.
